tests/tasks/setup_test.yml

```diff
@@ -0,0 +1,22 @@
+---
+- name: Set platform/version specific variables
+  include_role:
+    name: linux-system-roles.vpn
+    tasks_from: set_vars
+    public: true
+
+- name: Change inventory_hostname
+  set_fact:
+    inventory_hostname={{ main_hostname }}
```

Comment:
don't you need to use yaml syntax here?  `inventory_hostname: "{{ main_hostname }}"`

Reply:
I think you can set strings and boolean values using the `var={{ value }}` notation as well. It's mentioned in the notes section of the Ansible documentation [https://docs.ansible.com/ansible/latest/collections/ansible/builtin/set_fact_module.html#notes](https://docs.ansible.com/ansible/latest/collections/ansible/builtin/set_fact_module.html#notes). But for the sake of consistency it's probably better to use the yaml syntax, so I'll go ahead and change it anyway